plugins/README.md

```diff
@@ -0,0 +1,58 @@
+Plugins API
+====
+
+The plugins API contains the following methods:
+
+### `wp.plugins.__experimental.registerPlugin( name: string, { render: function } )`
+
+This method takes two arguments:
+1. `name`: A string identifying the plugin. Must be unique across all registered plugins.
+2. `settings`: An object containing the following data:
+   - `render`: A component containing the UI elements to be rendered. See the list below for all available UI elements.
+  
+**Example**
+
+```jsx
+const { Fragment } = wp.element;
+const { PluginSidebar } = wp.editPost.__experimental;
+
+const Component = () => (
+    <Fragment>
+        <PluginSidebar name="first-sidebar-name" title="My Sidebar">
+            <p>Content of the first sidebar</p>
+        </PluginSidebar>
+        <PluginSidebar name="second-sidebar-name" title="My Second Sidebar">
+            <p>Content of the second sidebar</p>
+        </PluginSidebar>
+    </Fragment>
+);
+
+wp.plugins.__experimental.registerPlugin( 'plugin-names', {
```

Comment:
This is outdated, `wp.plugins.registerPlugin` is available. (PR description is also outdated.)

Reply:
>This is outdated, `wp.plugins.registerPlugin` is available. (PR description is also outdated.)

I missed this. I assumed we were still exporting as `__experimental`. I'm fine with "unexperimental"-izing it at this point.

Reply:
>This is outdated, `wp.plugins.registerPlugin` is available. (PR description is also outdated.)

Updated in 853655c.